src/SprykerEco/Zed/ComputopShipment/Business/QuoteShipmentExpander/QuoteDefaultShipmentExpander.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\ComputopShipment\Business\QuoteShipmentExpander;
+
+use Generated\Shared\Transfer\AddressTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Generated\Shared\Transfer\ShipmentMethodsTransfer;
+use Generated\Shared\Transfer\ShipmentMethodTransfer;
+use Generated\Shared\Transfer\ShipmentTransfer;
+use SprykerEco\Zed\ComputopShipment\Business\ComputopShipmentException;
+use SprykerEco\Zed\ComputopShipment\ComputopShipmentConfig;
+use SprykerEco\Zed\ComputopShipment\Dependency\ComputopShipmentToShipmentFacadeInterface;
+
+class QuoteDefaultShipmentExpander implements QuoteShipmentExpanderInterface
+{
+    /**
+     * @var \SprykerEco\Zed\ComputopShipment\ComputopShipmentConfig
+     */
+    protected $computopShipmentConfig;
+
+    /**
+     * @var \SprykerEco\Zed\ComputopShipment\Dependency\ComputopShipmentToShipmentFacadeInterface
+     */
+    protected $shipmentFacade;
+
+    /**
+     * @param \SprykerEco\Zed\ComputopShipment\ComputopShipmentConfig $computopShipmentConfig
+     * @param \SprykerEco\Zed\ComputopShipment\Dependency\ComputopShipmentToShipmentFacadeInterface $shipmentFacade
+     */
+    public function __construct(ComputopShipmentConfig $computopShipmentConfig, ComputopShipmentToShipmentFacadeInterface $shipmentFacade)
+    {
+        $this->computopShipmentConfig = $computopShipmentConfig;
+        $this->shipmentFacade = $shipmentFacade;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @throws \SprykerEco\Zed\ComputopShipment\Business\ComputopShipmentException
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    public function expand(QuoteTransfer $quoteTransfer): QuoteTransfer
+    {
+        $quoteTransfer->setDefaultShipmentSelected(true);
+
+        $defaultShipmentMethodKey = $this->computopShipmentConfig->getDefaultShipmentMethodKey();
+        $defaultShipmentMethodTransfer = $this->shipmentFacade->findShipmentMethodByKey($defaultShipmentMethodKey);
+        if ($defaultShipmentMethodTransfer === null || $defaultShipmentMethodTransfer->getIsActive() === false) {
+            throw new ComputopShipmentException('Default shipment method is not available!');
```

Comment:
It would be better for the developer that will support this code to have a actual shipping method key in the exception.